starboard/shared/win32/posix_emu/include/arpa/inet.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_SHARED_WIN32_POSIX_EMU_INCLUDE_ARPA_INET_H_
+#define STARBOARD_SHARED_WIN32_POSIX_EMU_INCLUDE_ARPA_INET_H_
+
+#if defined(STARBOARD)
```

Comment:
Should we add an defined(STARBOARD) && defined(_MSVC) here or not ? 

`_byteswap_foo()` doesn't exist on anything than a MS  compiler of course, and i know this is already explicitly in a win32/ directory. But i think it's helpful for readability to see the intent in the code directly that this is only for MSVC, and also provides error feedback if this file for any reason ever gets accidentally on the include paths for anything non-msvc.